crates/forge/src/scarb/config.rs

```diff
@@ -31,24 +30,21 @@ pub struct ForgeConfigFromScarb {
 
 #[derive(Debug, PartialEq, Clone)]
 pub struct ForkTarget {
-    name: String,
-    params: RawForkParams,
+    pub name: String,
+    pub url: String,
+    pub block_id_type: String,
+    pub block_id_value: String,
 }
 
 impl ForkTarget {
     #[must_use]
-    pub fn new(name: String, params: RawForkParams) -> Self {
-        Self { name, params }
-    }
-
-    #[must_use]
-    pub fn name(&self) -> &str {
-        &self.name
-    }
-
-    #[must_use]
-    pub fn params(&self) -> &RawForkParams {
-        &self.params
+    pub fn new(name: String, url: String, block_id_type: String, block_id_value: String) -> Self {
+        Self {
+            name,
+            url,
+            block_id_type,
+            block_id_value,
+        }
```

Comment:
What's preventing the developers from instantiating `ForkTarget` directly now that all it's fields are public? I think we could at least add `#[non_exhaustive]` to `ForkTarget`, so it cannot be created directly.

Reply:
non exhaustive has no effect btw https://doc.rust-lang.org/reference/attributes/type_system.html#the-non_exhaustive-attribute

Reply:
Is it worth to add then?

Reply:
It does not have an effect only within defining crate, with others it works. I think we can add it, it doesn't require much effort. But you can add an issue for that and add a todo for future.